sphinx/templates/imgmath/preview.tex_t

```diff
@@ -1,5 +1,5 @@
 \documentclass[12pt]{article}
-\usepackage[utf8x]{inputenc}
+\usepackage[utf8]{inputenc}
```

Comment:
Actually my explanations on why utf8x is strange apply to utf8 itself also : for rendering any non-ascii character LaTeX (except xelatex, lualatex) needs suitable specification of font encoding; as here there is no such specification in this template, the `\usepackage[utf8]{inputenc}` has the same issues I described, AFAICT, of `\usepackage[utf8x]{inputenc}`, but at least it avoids extra rather big dependency (`ucs` package).

Reply:
(sorry for never ending review). A `\usepackage[utf8]{inputenc}` is not useless because it allows input of European letters such as `\text{à}` in math mode. Indeed LaTeX will know how to convert this into using some accent as in ``\`a`` primitive TeX mark-up. But as explained prior more exotic non-ascii letters will not go through in general.

Reply:
I'm not sure why it is there. It was actually put in the predecessor of imgmath, i.e., pngmath: c41f516ddaa3b083e5aa5a5c982b2a7cc4819eb7

@birkenfeld is probably the right person to ask.

Reply:
Thanks @xuhdev. I don't see what `utf8x` brought even back then, perhaps `utf8` from LaTeX2e team had some limitations, but as I explained input encoding in pdfLaTeX can only work if matched to suitable font encodings. Perhaps @birkenfeld will comment later on.